src/mlpack/methods/decision_tree/decision_tree_regressor_impl.hpp

```diff
@@ -206,14 +205,14 @@ DecisionTreeRegressor<FitnessFunction,
     const size_t minimumLeafSize,
     const double minimumGainSplit,
     const std::enable_if_t<arma::is_arma_type<
-        typename std::remove_reference<WeightsType>::type>::value>*):
+        std::remove_reference_t<WeightsType>>::value>*):
```

Comment:
```suggestion
        std::remove_reference_t<WeightsType>>::value>*) :
```

Reply:
Done and also made the indentation of the following lines more consistent with the rest of the code.